docs/man_pages/lib-management/library-add.md

```diff
@@ -7,21 +7,23 @@ General | `$ tns library add <Platform> <Library Path>`
 
 Adds a locally stored native library to the current project. <% if(isHtml) { %>Copies the library files to the `lib/<platform>` folder in your project and configures the platform-specific projects in `platforms/<platform>` to work with the library. Build operations might perform additional configuration changes on the platform-specific project in `platforms/<platform>`.<% } %>
 
+IMPORTANT: The `tns library add` command is deprecated and will be removed in a future release. Use the plugin set of commands instead. For more information, run `tns help plugin`.
```

Comment:
Here, we can make this more user-friendly. For the "for more information part", use context.

```
For more information, <% if(isHtml) { %>see the [plugin](/plugin.html) set of commands.<% } %><% if(isConsole) { %>run `tns help plugin`.<% } %>
```